src/Security/Member.php

```diff
@@ -1188,15 +1188,15 @@ public static function map_in_groups($groups = null)
      *
      * If no groups are passed, all groups with CMS permissions will be used.
      *
-     * @param array $groups Groups to consider or NULL to use all groups with
+     * @param SS_List|array|null $groups Groups to consider or NULL to use all groups with
      *                      CMS permissions.
      * @return Map Returns a map of all members in the groups given that
      *                have CMS permissions.
      */
-    public static function mapInCMSGroups($groups = null)
+    public static function mapInCMSGroups(SS_List|array|null $groups = null): Map
     {
         // non-countable $groups will issue a warning when using count() in PHP 7.2+
-        if (!$groups) {
+        if ($groups === null) {
```

Comment:
Now that the param is typehinted we can be more specific, which will reduce errors.